model/src/main/scala/org/broadinstitute/dsde/rawls/model/RawlsEnumeration.scala

```diff
@@ -1,6 +1,6 @@
 package org.broadinstitute.dsde.rawls.model
 
-trait RawlsEnumeration[T <: RawlsEnumeration[T]] { self: T =>
+trait RawlsEnumeration[T <: RawlsEnumeration[T]] extends Product with Serializable { self: T =>
```

Comment:
Good question, type inference! Basically all case classes automatically extend Product and Serializable, so to avoid it showing up in inferred types (i.e. when using case classes with parent traits in collections) you have to add it to the base trait.

I could have also added it to the WorkflowStatus trait, but I'm kind of assuming that all concrete implementations of RawlsEnumeration will be case classes/objects.

This SO article helps explain it a little better: http://stackoverflow.com/questions/36525804/scala-case-class-extending-product-with-serializable